hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/dispatcher/ReadableStoreFactory.java

```diff
@@ -58,6 +60,8 @@ public class ReadableStoreFactory {
     // e.g. services have to register their stores
     private static final Map<Class<?>, StoreEntry> STORE_FACTORY = Map.of(
             ReadableAccountStore.class, new StoreEntry(TokenService.NAME, ReadableAccountStoreImpl::new),
+            ReadableNftStore.class, new StoreEntry(TokenService.NAME, ReadableAccountStoreImpl::new),
+            ReadableStakingInfoStore.class, new StoreEntry(TokenService.NAME, ReadableAccountStoreImpl::new),
```

Comment:
I assume `ReadableAccountStoreImpl` implements the `ReadableNftStore` and `ReadableStakingInfoStore` interfaces

Reply:
Good catch! This was probably a copy-and-paste error. Fixed it and added tests.